asm/banks/c5.asm

```diff
@@ -5,7 +5,14 @@ hirom
 ; -------------------------------------------------------------------------
 ; Expands the total number of songs the game can use and adds a pointer
 ; to the new song
+;
+; TODO: I don't think this is used anymore?
```

Comment:
I think the conflict with `master` is due to this (broken) code having been removed. In fact, the entire `c5.asm` bank no longer exists in `master`. IIFC, the code here breaks the `Silence` music track, which was the cause of lag when entering the Rich Man's house in South Figaro in a previous version.